tests/integration/tasks/test-address-list.yml

```diff
@@ -1,39 +1,34 @@
 ---
-- name: ALWAYS_CHANGES Test adding a firewall address-list
-  mt_ip_firewall_addresslist:
+- name: Test adding a firewall address-list
+  mt_ip:
     hostname:   "{{ mt_hostname }}"
     username:   "{{ mt_user }}"
     password:   "{{ mt_pass }}"
     state:      "present"
-    list_name: test_list
-    address_list:
-      - address: 192.168.1.2
-        comment: dns1
-      - address: 192.168.1.3
-        comment: dns2
-      - address: 192.168.1.6
-        comment: test_comment3
-  register: address_list_add_1
-  failed_when: (
-      not ansible_check_mode
-    ) and (
-      not ( address_list_add_1 | changed )
-    )
+    parameter: address-list
```

Comment:
this is a little confusing.  

There is no submodule called 'address-list' under the IP namespace.  address-list lives under firewall, which lives under IP.  What does the actual api call look like?